CHANGELOG.md

```diff
@@ -39,6 +39,7 @@
 * [FEATURE] Ruler: Add experimental support for caching the contents of rule groups. This is disabled by default and can be enabled by setting `-ruler-storage.cache.rule-group-enabled`. #9595
 * [FEATURE] PromQL: Add experimental `info` function. Experimental functions are disabled by default, but can be enabled setting `-querier.promql-experimental-functions-enabled=true` in the query-frontend and querier. #9879
 * [FEATURE] Distributor: Support promotion of OTel resource attributes to labels. #8271
+* [ENHANCEMENT] Query Frontend: Update Server-Timing `bytes_processed` format #9985
```

Comment:
since this is a change on top of another change that's not released yet, can you update the existing changelog entry from the previous PR?

> Return server-side total bytes processed statistics as a header through query frontend

(maybe also update it so it's more clear what's changing - the name of the header and the name of the value; as it is, it's barely informative)